### DISCUSSION TITLE
COVID-19 codelist (icd10)

### DISCUSSION THREAD

**Discussion by qwenww** [opensafely/documentation#1480] [Codelists] [answered]
Hi, I would like to know when to use three items ["U071", "U072", "U109"] for COVID-19 codelist instead of usual two items ["U071", "U072"]?  (U109: Multisystem inflammatory syndrome associated with COVID-19, unspecified), thanks!

**Comment by wjchulme** [comment, marked as answer]
Reposting a digested answer I wrote in slack a while ago:

## Abstract

Since Feb 2021, two more ICD-10 codes for Covid, `U099` and `U109`, have been added on top of the usual `U071` and `U072`. These are used by ONS when reporting Covid-related deaths. See attached image for description, taken from [this ONS page](https://www.ons.gov.uk/peoplepopulationandcommunity/birthsdeathsandmarriages/deaths/methodologies/userguidetomortalitystatisticsjuly2017). ht @cewalters 

![image (2)](https://github.com/opensafely/documentation/assets/25637345/d5ca4d74-0f74-4c8f-a1c9-721f7deebdaf)

To investigate how these ICD-10 codes were being used, we looked at covid-related unplanned hospital admissions and deaths, using different combinations of the codes.

tl;dr -- The new codes appear to never be used for hospital admissions, and only very rarely for death certification. Old studies that didn't include the new codes shouldn't need to be revisited. But it's still worth including them for future studies where relevant.

## Methods

We looked at these codes when appearing as primary reason for admission or cause of death, and when appearing as any reason.
We used a simplified version of the comparative booster effectiveness population -- basically, anyone vaccinated with Moderna or Pfizer from late october 2021 to january 2022.

We looked a time to covid-related hospitalisation or death following booster vaccination, broken down by different code combinations, eg `U071` only, `U072` only, `U071;U099`, etc

## Results

Hospitalisations: No new codes (`U099`, `U109`) appeared in coding for hospitalisation. The vast majority were `U071`-coded events. The rest were `U072` or `U071;U072`.

Deaths: A small number of deaths (<50) included the new code `U099`, but most of these were accompanied by `U071`. Very few (<10) covid-related deaths would be missed if `U099` was not used to in the identifying codelist. `U109` was also used very few times (<10) times, always accompanied by another `U071` code.

## Notes

Code use may be different in unvaccinated / undervaccinated populations. In particular `U109` (multisystem organ failure associated with covid-19) might be more likely to appear, as severe events are more common.

The breakdown of code use considered only the first event containing at least one of the codes after booster vaccination. This means, for example, that a person admitted to hospital with covid under code `U071` (confirmed covid), who was later admitted to critical care under code `U109` (multisystem organ failure associated with covid-19) will not have been picked up in the summary, because admission to critical care usually means a new hospital spell, and so wouldn't be the first event after vaccination. However, it's possible that the new codes `U099`, `U109` simply aren't available for use when coding hospital events.
